### ISSUE TITLE
Is anyone successfully train Lora on Z base image ?

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#908]
Hi there, trying since few days to train my lora's on z base image model with both Adafactor ( similar settings from the readme ) + also tried prodigy but the accuracy is way way off compare to others models, is anyone get good result with musubi and z base image ? Feel like only onetrainer is able to train correctly this model.

Thanks

**Comment by Toserme** [comment]
I completed my second ZIB Lora run today and results are pretty much perfect. Launch commands I used (without file paths)

--num_cpu_threads_per_process 1
--mixed_precision bf16 src/musubi_tuner/zimage_train_network.py
--network_module networks.lora_zimage
--sdpa
--mixed_precision "bf16"
--timestep_sampling sigmoid
--discrete_flow_shift 1.0
--learning_rate 1e-4
--xformers
--lr_scheduler "constant_with_warmup"
--lr_warmup_steps 100
--optimizer_type "adamw"
--optimizer_args "weight_decay=0.01" "betas=(0.9, 0.99)"
--max_grad_norm 1
--network_dropout 0.1
--gradient_checkpointing
--max_train_epochs 40
--network_dim 32
--network_alpha 32

Dataset was around 40 images at 768 training res with 3 repeat and batch size 4. 

Lora works without issues at 1.0 str. Even with GGUF model and distill Loras. Does not work well with ZIT tho.

**Comment by BenDes21** [comment]
> I completed my second ZIB Lora run today and results are pretty much perfect. Launch commands I used (without file paths)
> 
> --num_cpu_threads_per_process 1 --mixed_precision bf16 src/musubi_tuner/zimage_train_network.py --network_module networks.lora_zimage --sdpa --mixed_precision "bf16" --timestep_sampling sigmoid --discrete_flow_shift 1.0 --learning_rate 1e-4 --xformers --lr_scheduler "constant_with_warmup" --lr_warmup_steps 100 --optimizer_type "adamw" --optimizer_args "weight_decay=0.01" "betas=(0.9, 0.99)" --max_grad_norm 1 --network_dropout 0.1 --gradient_checkpointing --max_train_epochs 40 --network_dim 32 --network_alpha 32
> 
> Dataset was around 40 images at 768 training res with 3 repeat and batch size 4.
> 
> Lora works without issues at 1.0 str. Even with GGUF model and distill Loras. Does not work well with ZIT tho.

Hello there ! Thanks a lot, gonna give a try and let you know, :D 

For the dataset.toml do you let : enable_bucket = true
bucket_no_upscale = false ? 

Thanks

**Comment by Toserme** [comment]
> > I completed my second ZIB Lora run today and results are pretty much perfect. Launch commands I used (without file paths)
> > --num_cpu_threads_per_process 1 --mixed_precision bf16 src/musubi_tuner/zimage_train_network.py --network_module networks.lora_zimage --sdpa --mixed_precision "bf16" --timestep_sampling sigmoid --discrete_flow_shift 1.0 --learning_rate 1e-4 --xformers --lr_scheduler "constant_with_warmup" --lr_warmup_steps 100 --optimizer_type "adamw" --optimizer_args "weight_decay=0.01" "betas=(0.9, 0.99)" --max_grad_norm 1 --network_dropout 0.1 --gradient_checkpointing --max_train_epochs 40 --network_dim 32 --network_alpha 32
> > Dataset was around 40 images at 768 training res with 3 repeat and batch size 4.
> > Lora works without issues at 1.0 str. Even with GGUF model and distill Loras. Does not work well with ZIT tho.
> 
> Hello there ! Thanks a lot, gonna give a try and let you know, :D
> 
> For the dataset.toml do you let : enable_bucket = true bucket_no_upscale = false ?
> 
> Thanks

bucket_no_upscale = true

You dont want your images to be upscaled by the trainer. Just make sure your images are higher resolution than your training res (in my case 768x768).

I initially created my dataset for 1024x1024 resolution but I was OOMing so I set the training res to 768 and the bucketing took care of the rest.

One more thing - makes sure to convert your trained loras using convert_lora.py script inside musubi folder if you are going to use it with ComfyUI.

**Comment by Winterium99** [comment]
> I completed my second ZIB Lora run today and results are pretty much perfect. Launch commands I used (without file paths)
> 
> --num_cpu_threads_per_process 1 --mixed_precision bf16 src/musubi_tuner/zimage_train_network.py --network_module networks.lora_zimage --sdpa --mixed_precision "bf16" --timestep_sampling sigmoid --discrete_flow_shift 1.0 --learning_rate 1e-4 --xformers --lr_scheduler "constant_with_warmup" --lr_warmup_steps 100 --optimizer_type "adamw" --optimizer_args "weight_decay=0.01" "betas=(0.9, 0.99)" --max_grad_norm 1 --network_dropout 0.1 --gradient_checkpointing --max_train_epochs 40 --network_dim 32 --network_alpha 32
> 
> Dataset was around 40 images at 768 training res with 3 repeat and batch size 4.
> 
> Lora works without issues at 1.0 str. Even with GGUF model and distill Loras. Does not work well with ZIT tho.

hello, can i ask what tool did you use to train the z imaghe base lora? i have kohya ss and i dont know if it ll work
